src/engine.jl

```diff
@@ -15,8 +15,8 @@ type MSession
     buffer::Vector{UInt8}
     bufptr::Ptr{UInt8}
 
-    function MSession(bufsize::Integer = default_output_buffer_size)
-        ep = ccall(engfunc(:engOpen), Ptr{Void}, (Ptr{UInt8},), default_startcmd)
+    function MSession(bufsize::Integer = default_output_buffer_size, options::String = " -nosplash")
```

Comment:
It's fine to open additional PR's that includes a previous PR. but if you don't stop adding things to a PR the review process will never end.